doc/vector-extra/riscv-crypto-spec-vector-extra.adoc

```diff
@@ -0,0 +1,159 @@
+[[riscv-doc-template]]
+= RISC-V Cryptography Extensions Volume III: Extra Vector Instructions
+:description: The vector extra cryptography extensions for the RISC-V ISA.
+:company: RISC-V.org
+:revdate: 31 August 2023
+:revnumber: v0.0.1
+:revremark:
+:url-riscv: http://riscv.org
+:doctype: book
+//:doctype: report
+:preface-title: Preamble
+:colophon:
+:appendix-caption: Appendix
+:imagesdir: images
+:title-logo-image: image:risc-v_logo.png[pdfwidth=3.25in,align=center]
+//:page-background-image: image:draft.svg[opacity=20%]
+//:title-page-background-image: none
+//:back-cover-image: image:circuit.png[opacity=25%]
+// Settings:
+:experimental:
+:reproducible:
+// needs to be changed? bug discussion started
+:WaveDromEditorApp: wavedrom-cli
+:imagesoutdir: images
+:icons: font
+:lang: en
+:listing-caption: Listing
+:sectnums:
+:toc: left
+:toclevels: 4
+:source-highlighter: pygments
+ifdef::backend-pdf[]
+:source-highlighter: coderay
+endif::[]
+:data-uri:
+:hide-uri-scheme:
+:stem: latexmath
+:footnote:
+:xrefstyle: short
+:bibtex-file: riscv-crypto-spec.bib
+:bibtex-order: alphabetical
+:bibtex-style: ieee
+
+//:This is the preamble.
+
+[colophon]
+= Colophon
+
+This document describes the Vector Cryptography Extra extensions to the
```

Comment:
`Zfa` is using "additional" which may be better here (https://github.com/riscv/riscv-isa-manual/blob/main/src/zfa.adoc).

Reply:
That document is titled ""Zfa" Standard Extension for Additional Floating-Point Instructions, Version 1.0".  That's less ambiguous because it includes the actual extension code Zfa, which won't be reused by any future extension.  This document just refers to "Vector Cryptography Extra extensions" as if there will never be any more.  Maybe include the actual extension codes in the title?

Reply:
Good idea, I will update the title to manage room for future "extra"/additional extensions.